packages/fuel-gauge/scripts/build.sh

```diff
@@ -6,7 +6,7 @@ for PROJECT_DIR in ./test-projects/* ;
 do
   BIN_DIR="$PROJECT_DIR/out/debug"
 
-  ./../../packages/forc-bin/forc-binaries/forc build -p $PROJECT_DIR
+  pnpm forc build -p $PROJECT_DIR
```

Comment:
This is the PR that changed this to use an absolute path instead:
 - https://github.com/FuelLabs/fuels-ts/pull/557/files#r1040574105

Since then, we've been having problems again:
 - https://github.com/FuelLabs/fuels-ts/pull/763#issuecomment-1424169679

Also, there have been other situations (under Linux) where `pnpm` wasn't finding the correct binary.

The fix was again to use the full path:
- https://github.com/FuelLabs/fuels-ts/pull/772

TBH, I don't know what's happening, but it looks like a delicate change.

Reply:
On my local, the path was not working... but with pnpm it worked. And it looks to be the correct way to do this. If pnpm fails on every Linux, it shall fail on CI also. This looks like to be a machine-specific issue and the solution is related to pnpm not to the project it self.